src/opnsense/scripts/wireguard/wg-service-control.php

```diff
@@ -309,6 +318,9 @@ interfaces_addresses_flush((string)$node->interface, 6, $ifdetails);
 
                             mwexecf('/sbin/ifconfig %s %s', [$node->interface, $carp_if_flag]);
                         }
+
+                        wg_apply_debug_flag($node->interface->getValue(), $node->debug->getValue());
```

Comment:
logically this shouldn't be needed, unless down/up removes the flag for some vague reason.